src/inc/LibraryIncludes.h

```diff
@@ -65,8 +65,11 @@
 // GSL
 // Block GSL Multi Span include because it both has C++17 deprecated iterators
 // and uses the C-namespaced "max" which conflicts with Windows definitions.
-#include <gsl/gsl>
+#if __has_include(<gsl/gsl_util>)

```

Comment:
GSL currently no longer has a header file called gsl_util but has utils.

Reply:
FWIW my goal was to remove the GSL dependency entirely from the project in the near term, so this change may not be necessary since we aren't planning to update it either.